src/Microsoft.ML.PCA/PcaTrainer.cs

```diff
@@ -469,6 +470,20 @@ public void SaveAsText(TextWriter writer, RoleMappedSchema schema)
             }
         }
 
+        public IDataView GetSummaryDataView(RoleMappedSchema schema)
+        {
+            var bldr = new ArrayDataViewBuilder(Host);
+
+            bldr.AddColumn("Mean vector", NumberType.R4, _mean);
```

Comment:
>Mean vector [](start = 28, length = 11)

While not explicitly proscribed, elsewhere we have held to the convention that these names ought to be valid C# identifiers... which is to say, `PascalCased`.